CLAUDE.md

```
@@ -55,7 +56,8 @@
 ```
 
 ### iOS SDK Development
-
+## While implementing:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)